source/release-notes.txt

```diff
@@ -11,6 +11,46 @@ Release Notes
    :class: twocols
 
 
+1.10.0 Changelog
+---------------
+
+*Released October 1, 2024*
+
+New features:
+
+- New bulk conversion UI and API for Query Converter to automatically 
+  convert large numbers of SQL queries, views and stored procedures. 
+  See :ref:`rm-bulk-conversion`.
```

Comment:
(question)
This will resolve once that branch is merged. Is that right?

Reply:
That's correct! I just merged the bulk conversions working PR into the feature branch.